packages/client/src/pages/Home/components/Rules/rules.module.css

```
@@ -0,0 +1,105 @@
+.containerRules {
+  background-color: rgb(244, 234, 169);
+  border-radius: 40px 40px 0 0;
+}
+
+.dividers {
+  display: flex;
+  justify-content: space-between;
+}
+
+.shortDivider,
+.longDivider {
+  border: 2px solid rgb(32, 155, 25);
+}
+
+.shortDivider,
+.subtitle {
+  width: 22%;
+}
+
+.longDivider,
+.title,
+.startImage {
+  width: 72%;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  gap: 60px;
+}
+
+.titlesContainer {
+  display: flex;
+  margin-top: 36px;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.subtitle,
+.title {
+  color: rgb(32, 155, 25);
+}
+
+.subtitle {
+  font-weight: 700;
+}
+
+.title {
+  font-size: 100px;
```

Тоже миксин можно использовать
```suggestion
  composes: font-design-headline-l from global;
```